main/Card/index.tsx

```diff
@@ -0,0 +1,136 @@
+import {
+  ActivityIndicator,
+  ImageSourcePropType,
+  ImageStyle,
+  TextStyle,
+  ViewStyle,
+} from 'react-native';
+import React, { FC, ReactNode, ReactNodeArray } from 'react';
+
+import { FlattenSimpleInterpolation } from 'styled-components';
+import styled from 'styled-components/native';
+
+import styles from './styles';
+
+interface IContainerProps {
```

Comment:
Just use `ContainerProps` without prefix I.

Reply:
Refs: https://github.com/microsoft/TypeScript-Handbook/issues/121#issuecomment-412810455

Reply:
thk u for comment!
i resolved that in 2b761a7